models/sipnet/R/read_restart.SIPNET.R

```diff
@@ -78,14 +73,19 @@ read_restart.SIPNET <- function(outdir, runid, stop.time, settings, var.names, p
     names(forecast[[length(forecast)]]) <- c("LeafC")
   }
   
+  if ("LAI" %in% var.names) {
+    forecast[[length(forecast) + 1]] <- ens$LAI[last]  ## m2/m2 
+    names(forecast[[length(forecast)]]) <- c("LAI")
+  }
+  
   if ("litter_carbon_content" %in% var.names) {
     forecast[[length(forecast) + 1]] <- ens$litter_carbon_content[last]  ##kgC/m2
     names(forecast[[length(forecast)]]) <- c("Litter")
   }
-  
-  if ("TotSoilCarb" %in% var.names) {
-    forecast[[length(forecast) + 1]] <- ens$TotSoilCarb[last]  ## kgC/m2
-    names(forecast[[length(forecast)]]) <- c("TotSoilCarb")
+
+  if ("NEE" %in% var.names) {
+    forecast[[length(forecast) + 1]] <- ens$NEE[last]  ## gC/m2
+    names(forecast[[length(forecast)]]) <- c("NEE")
```

Comment:
NEE already exists earlier in the file. Not sure why there's a duplicate

Reply:
Removed duplicate

Reply:
Removed duplicate